packages/react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -425,6 +426,7 @@ export class Suggestions<T> extends React.Component<ISuggestionsProps<T>, ISugge
               removeButtonAriaLabel={removeSuggestionAriaLabel}
               onRemoveItem={this._onRemoveTypedSuggestionsItem(suggestion.item, index)}
               id={'sug-' + index}
+              removeButtonIconProps={removeButtonIconProps ?? { iconName: 'Cancel' }}
```

Comment:
(From my previous review): You shouldn't default this here; let SuggestionsItem provide the default internally.
```suggestion
              removeButtonIconProps={removeButtonIconProps}
```